### PR TITLE
Log queue consumer and message counts along with timings for messages

### PR DESCRIPTION
**Pull request by ianmcorvidae**
This is a relatively small thing which can make it easier to create visualizations (particularly ones comparing number of consumers vs. time to consume, or number of messages vs. number of messages consumed, etc.). I've added these fields to the same message that already has the timing information.

*NOTE*: it'll probably be easier for all of us if we add this to the ES templates/mappings before this is merged and deployed anywhere.

### CODE REVIEW

File: src/dewey/curation.clj [hunk 1]
```diff
@@ -348,6 +350,10 @@
               (irods-backoff)) ; This will retry forever
             (throw e)))
         (log/warn (str "unknown routing key" routing-key "received with message" msg)))
-      (let [consumetime (milliseconds-since consume-start)]
-        (tc/with-logging-context {:amqp-total-time consumetime :dewey-consume-time @innertime}
-          (log/info (format "[curation/consume-msg] [%s] [%s] %gms consume/%gms total" routing-key msg @innertime consumetime)))))))
+      (let [consumetime (milliseconds-since consume-start)
+            queue-stats (lq/status channel (cfg/queue-name))]
+        (tc/with-logging-context {:amqp-message-count (:message-count queue-stats)
+                                  :amqp-consumer-count (:consumer-count queue-stats)
```
Comment (ianmcorvidae) [inline src/dewey/curation.clj:356]: I'm not sure if these should include "queue" somewhere in their name, to be (potentially) clearer that they refer to one specific queue, or some other specifier.

Comment (strootman) [reply src/dewey/curation.clj:356]: Good call.
For now, I don't think it completely necessary. 
In the future, we could come up with some form of nested mapping, to capture "per queue" stats.

**Comment by strootman** [comment]
I see no issues with the names.
I'll report back here when the mappings are in place.

**Comment by ianmcorvidae** [comment]
Excellent, thanks for the review. I'll wait on your word.

**Review by strootman — APPROVED** [review]
Current index mappings as well as index template have been updated with new mappings for `amqp-message-count` and `amqp-consumer-count`. Both keys are of type `integer`, determined by inspecting `com.rabbitmq.client.AMQP` class via _langohr_.

**Pull request merged**